src/cli/command-registrar.ts

```diff
@@ -232,9 +233,16 @@ export default class CommandRegistrar {
     ) {
       process.stdout.write(
         chalk.yellow(
-          `warning: '${chalk.bold(subcommand)}' is not a valid command.\nsee 'bit --help' for additional information.\n`
+          ` warning: '${chalk.bold(
+            subcommand
+          )}' is not a valid command.\nsee 'bit --help' for additional information.\n`
         )
       );
+      const suggestion = didYouMean(subcommand, commander.commands.map(cmd => cmd._name));
```

Comment:
please filter commands with the `private = true` property. for example, see: https://github.com/teambit/bit/blob/master/src/cli/commands/private-cmds/_fetch-cmd.ts#L11

Reply:
Please clarify.

Reply:
Some of the commands we have are not visible by the help and not fully documented since they are used for debugging / internal use / used only on a bit server.
I don't think we want to suggest those commands (at least not the ones with private=true property which are only running on a server so they never typed by a human)